playbooks/sync-task.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Configure instance(s)
+  hosts: all
+  become: True
+  gather_facts: No
```

Comment:
@Metfriet, this should be removed because gathering of facts is necessary for populating the built-in variables like `ansible_distribution_release` etc.